pkg/api/v1/conversion.go

```diff
@@ -125,7 +125,9 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name":
```

Comment:
I think name is used in v1beta1, v1beta2, v1beta3. but we deprecated them all. And I found that other kinds also had name label before, but it was removed. I don't know why "Namespace",  "PersistentVolume" and "PersistentVolumeClaim" should kept it. Maybe @kubernetes/goog-csi could give us a clear answer. and decide whether we should support it in v1?